crates/core/src/factories/mem_op_store/test.rs

```diff
@@ -1,8 +1,16 @@
 use crate::factories::mem_op_store::Kitsune2MemoryOpStore;
 use crate::factories::MemoryOp;
-use kitsune2_api::{DynOpStore, SpaceId, Timestamp};
+use kitsune2_api::{DhtArc, DynOpStore, SpaceId, Timestamp};
 use std::sync::Arc;
 
+fn op_at_location(loc: u32) -> MemoryOp {
+    let mut out = vec![];
+    out.extend_from_slice(&loc.to_le_bytes());
+    out.resize(32, 0);
```

Comment:
```suggestion
    let mut out = Vec::with_capacity(32);
    out.extend_from_slice(&loc.to_le_bytes());
    out.resize(32, 0);
```

Reply:
not that it really matters for a test : )

Reply:
Can't hurt to be responsible :)